smithy-openapi/src/main/java/software/amazon/smithy/openapi/OpenApiConfig.java

```diff
@@ -48,6 +48,16 @@ public enum HttpPrefixHeadersStrategy {
         WARN
     }
 
+    /** Specifies how to resolve multiple error responses with same error codes. */
+    public enum ErrorStatusConflictHandlingStrategy {
+        /** the default setting that uses OpenAPI's oneOf keyword to combine multiple schemas for same media type. */
+        USE_ONEOF,
```

Comment:
We used "ONE_OF" in other similar enums. Let's change to match. https://github.com/awslabs/smithy/blob/main/smithy-jsonschema/src/main/java/software/amazon/smithy/jsonschema/JsonSchemaConfig.java#L44

Reply:
resolved.